GVFS/GVFS.Common/IProcessHelper.cs

```diff
@@ -0,0 +1,12 @@
+namespace GVFS.Common
+{
+    /// <summary>
+    /// Interface around process helper methods. This is to enable
+    /// testing of components that interact with the ProcessHelper
+    /// static class.
+    /// </summary>
+    public interface IProcessHelper
```

Comment:
I would name this `IProcessRunner` or `IProcessLauncher`

Reply:
Renamed this to `IProcessRunner`